codeformat/formatter-config.xml

```diff
@@ -342,7 +342,7 @@
         <setting id="org.eclipse.jdt.core.formatter.keep_empty_array_initializer_on_one_line" value="false"/>
         <setting id="org.eclipse.jdt.core.formatter.alignment_for_assertion_message" value="0"/>
         <setting id="org.eclipse.jdt.core.formatter.insert_space_before_opening_paren_in_constructor_declaration" value="do not insert"/>
-        <setting id="org.eclipse.jdt.core.formatter.blank_lines_before_new_chunk" value="1"/>
+        <setting id="org.eclipse.jdt.core.formatter.blank_lines_before_new_chunk" value="0"/>
```

Comment:
Does this mean that the following will remain untouched?

```java
public void method() {} // No blank line after this method!
public void method2() {}
```

Reply:
No, that would still get a blank line in between, but sadly
```java
public void method() {}
public static class Class {}
```
would not as it seems